spark/src/main/scala/org/apache/comet/serde/QueryPlanSerde.scala

```diff
@@ -211,7 +211,7 @@ object QueryPlanSerde extends Logging with ShimQueryPlanSerde with CometExprShim
             // TODO add support for Count (this was removed when upgrading
             // to DataFusion 40 because it is no longer a built-in window function)
             // https://github.com/apache/datafusion-comet/issues/645
```

Comment:
This comment can be removed as part of this PR

```suggestion
```

Reply:
Removed. Thanks